### ISSUE TITLE
ember-cli@1.13.8   Using `Ember.HTMLBars.makeBoundHelper` is deprecated.

### ISSUE THREAD

**Issue by donovan-graham** [ember-cli/ember-cli#4871]
I've just updated to ember-cli 1.13.8 and am now getting this deprecation warning.
```
DEPRECATION: Using `Ember.HTMLBars.makeBoundHelper` is deprecated. Please refactor to using `Ember.Helper` or `Ember.Helper.helper`
```

I only have one helper named `to-dash`, which uses the compute syntax. 

```
import Ember from 'ember';

export default Ember.Helper.extend({
  compute(params) {
    if (params[0]) {
      let label = this._stream.params[0].label;
      let pos = label.lastIndexOf('.');
      if (pos !== -1) {
        label = label.substr(pos + 1);
      }
      return label.dasherize();
    }
  }
});
```

This is describe in the ember docs here:
http://emberjs.com/api/classes/Ember.Helper.html#method_compute

Please could you help me in understanding and fixing this deprecation message.

**Comment by stefanpenner** [comment]
For questions, Stackoverflow is a good venue. Or review embers deprecation guie: http://emberjs.com/deprecations/v1.x/

This issue tracker is for reported bugs with the build tooling, and this does not appear to be a bug with the build tooling

**Comment by donovan-graham** [comment]
I was able to fix this by bumping `ember-data` to `1.13.9` in package.json and bower.js.

It's a know bug in the build for ember-data 1.13.8 as documented here
https://github.com/emberjs/data/issues/3635#issuecomment-129498172

Maybe you could update ember-cli?
